Formula/libmagic.rb

```diff
@@ -5,6 +5,7 @@ class Libmagic < Formula
   sha256 "3751c7fba8dbc831cb8d7cc8aff21035459b8ce5155ef8b0880a27d028475f3b"
   # libmagic has a BSD-2-Clause-like license
   license :cannot_represent
+  revision 1
```

Comment:
```suggestion
```
I think we can drop this for now. We can add the `revision` if enough users complain about it.